catalog/transformer/catalog-transformer-csv-queryresponse/src/test/java/ddf/catalog/transformer/csv/CsvQueryResponseTransformerTest.java

```diff
@@ -119,6 +120,42 @@ public void testCsvQueryResponseTransformer() throws CatalogTransformerException
     assertThat(scanner.hasNext(), is(false));
   }
 
+  @Test
+  public void testCsvQueryResponseTransformerWithNoArgs() throws CatalogTransformerException {
+    Map<String, Serializable> argumentsMap = new HashMap<>();
+
+    assertThat(ATTRIBUTE_DESCRIPTOR_LIST.size(), is(ATTRIBUTE_DATA.length));
+
+    BinaryContent bc = transformer.transform(sourceResponse, argumentsMap);
+    Scanner scanner = new Scanner(bc.getInputStream());
+    scanner.useDelimiter("\\n|\\r|,");
+
+    /*
+     * OBJECT types, BINARY types and hidden attributes will be filtered out
```

Comment:
OBJECT types, BINARY types, hidden attributes and empty attributes (?)

Reply:
udpated